test/skimNtuple_HHbtag.cpp

```diff
@@ -3231,8 +3362,17 @@ int main (int argc, char** argv)
 				  int passCross = 1;
 				  int passSingle = 1;
 
-				  if (tlv_firstLepton.Pt() < 25.) passSingle = 0;
-				  if (tlv_secondLepton.Pt() < 32.) passCross = 0;
+				  float lep1_thresh, lep2_thresh;
```

Comment:
For 2016 there is no cross electron trigger son an additional if statement is required in the ETau section to avoid looking for it.